docs/articles/getting-started.md

```diff
@@ -1,37 +1,37 @@
----
-uid: getting-started
----
-
 # Getting Started
 
-## 1. How to install Colore
+***Important***: *This page is for v6.0.0 of Colore, which is currently only available as a pre-release. For the v5.2 getting started guide (not recommended), [[click here|Getting-started-(v5.2)]].*
```

Comment:
Not sure if this needs to be mentioned here. When these changes go live, 6.0 will be the stable release anyway.